Dockerfile

```diff
@@ -46,7 +46,16 @@ RUN case ${TARGETPLATFORM} in \
 
 # Copy binary to runtime image
 FROM ${RUNTIME_IMAGE}
+ARG VERSION
```

Comment:
Duplication. Already defined in line 22.

Please move the `ARG VERSION` from line 22 to the top below:

`ARG RUNTIME_IMAGE=gcr.io/distroless/static:nonroot`

Reply:
Sure thing, i'll try to get to that after work